### DISCUSSION TITLE
After updating to version 0.17.3 of RMM tactical, all hosts were in "overdue" status

### DISCUSSION THREAD

**Discussion by Cleberson-Brandao** [amidaware/tacticalrmm#1877] [General]
After updating to version 0.17.3 of RMM tactical, all hosts were in "overdue" status

**Comment by dinger1986** [comment]
current version is 18.2 so did you upgrade to that from 0.17.3?

**Reply by Cleberson-Brandao** [reply]
In fact, the version we currently have is 0.17.3, and it has this problem, even after restarting the host it still has this problem. 

Version 0.18.2 is still under evaluation, which is why I haven't performed the update yet.

**Reply by dinger1986** [reply]
evaluation of what? check your certs and hosts file
